rust/Trunk.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+target = "osm2lanes-web/index.html"
```

Comment:
I've never used trunk before, but wow, it's fantastic! I want to switch another huge project to using it too